docs/src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties.md

```diff
@@ -15,6 +15,11 @@ the affine and non-affine case:
     The lattice is always assumed to be the standard lattice $\mathbb{Z}^n$.
     Transformations for non-standard lattices will have to be done by the user.
 
+!!! warning
+```@docs
+    Base.:(==)(tv1::NormalToricVariety, tv2::NormalToricVariety)
+```
```

Comment:
Have you verified that this looks fine in a builded documentation
I am not sure that markdown understands here what you want to do

Reply:
Not yet... Actually, I wanted to circumvent this by writing things up a little different, so that I will not have to worry about this issue.